src/Sulu/Bundle/AdminBundle/Resources/js/containers/Form/fields/SmartContent.js

```diff
@@ -52,11 +65,27 @@ export default class SmartContent extends React.Component<Props> {
             formInspector.resourceKey === provider ? formInspector.id : undefined
         );
 
-        autorun(this.handleFilterCriteriaChange);
+        smartContentStorePool.add(this.smartContentStore, excludeDuplicates);
+
+        this.filterCriteriaChangeDisposer = autorun(this.handleFilterCriteriaChange);
+
+        if (!excludeDuplicates || this.previousSmartContentStores.length === 0) {
+            this.smartContentStore.start();
+        } else {
+            when(
+                () => this.previousSmartContentStores.every((store) => !store.itemsLoading),
+                (): void => {
+                    smartContentStorePool.updateExcludedIds();
```

Comment:
also this line will affect the `itemsLoading` property of the `previousSmartContentStores`, right?

Reply:
This only makes the previous smart content stores load something if one of the previous stores changed its content as well, which is not possible, unless the content manager changed something in the mean time on their own.